README.md

```diff
@@ -37,6 +31,77 @@ Here is a short preview of it looks like currently:
 A roadmap is available in the issue [#5](https://github.com/prometheus-community/codemirror-promql/issues/5).
 
 ## Usage
+
+As the setup of the promQL language can a bit tricky in CMN, this lib provides a class `PromQLExtension` 
+which is here to help you to configure the different extension we aim to provide.
+
+### Default setup
+If you want to enjoy about the different features provides without taking too much time to understand how to configure it,
+then the easiest way is this one:
+
+```typescript
+import { PromQLExtension } from 'codemirror-promql';
+import { basicSetup, EditorState, EditorView } from '@codemirror/next/basic-setup';
+
+const promQL = new PromQLExtension()
+new EditorView({
+      state: EditorState.create({
+        extensions: [basicSetup, promQL.asExtension()],
+      }),
+      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+      // tslint:disable-next-line:no-non-null-assertion
+      parent: document.getElementById('editor')!,
+    });
```

Comment:
Should this and lines above be indented less (only two spaces)?

Reply:
yeah you are right. Will fix it